frontend/src/components/common/header/header.tsx

```diff
@@ -1,27 +1,41 @@
+import { useState } from 'react';
+
 import { FC } from 'common/types/types';
 import { Image } from 'components/common/common';
+import { Popup } from './components/menu-popup/menu-popup';
 import logo from 'assets/img/logo.svg';
 import defaultUserAvatar from 'assets/img/avatar-default.svg';
-
 import styles from './styles.module.scss';
 
-const Header: FC = () => (
-  <header>
-    <div className={styles.headerWrapper}>
-      <div className={styles.logoWrapper}>
-        <Image width="150" height="94" src={logo} alt="logo" />
-      </div>
-      <div className={styles.userWrapper}>
-        <Image
-          width="50"
-          height="50"
-          src={defaultUserAvatar}
-          alt="user avatar"
-          isCircular
-        />
+const Header: FC = () => {
+  const [isMenuPopupVisible, setIsMenuPopupVisible] = useState(false);
```

Comment:
```suggestion
  const [isMenuPopupVisible, setIsMenuPopupVisible] = useState<boolean>(false);
```

Reply:
`useState(false)` automatically sets the type to `boolean`, what is the advantage of explicit typing here?

Reply:
always type `useState`